in/landuse/landuseclass.def

```diff
@@ -0,0 +1,9 @@
+1        Beets                     in/landuse/Beets.PAR
+2        Coniferous Forest         in/landuse/Coniferous_forest.PAR
+3        Corn                      in/landuse/Corn.PAR  
+4        Crops & Vegetables        in/landuse/Crops_vegetables.PAR
+5        Deciduous_forest          in/landuse/Deciduous_forest.PAR
+6        Indian Mustard            in/landuse/Indian_mustard.PAR
+7        Pasture                   in/landuse/Pasture.PAR
+8        Rapeseed + Field Mustard  in/landuse/Rapeseed_FieldMustard.PAR
```

Comment:
In Line 4 you used `&` for the combination of two crops, here you used a `+` sign, again, for consistency we should decide for one (I vote for `&`)